<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminConflictHandler</class>
 <widget class="QDialog" name="AdminConflictHandler">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>835</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Conflict Handler</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Icons.qrc">
    <normaloff>:/Icons/bulb14.png</normaloff>:/Icons/bulb14.png</iconset>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Item Being Removed : </string>
   </property>
  </widget>
  <widget class="QLabel" name="conflictItemLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>label</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>80</y>
     <width>256</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>60</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Please pick item to replace in conflicting schedules : </string>
   </property>
  </widget>
  <widget class="QPushButton" name="chooseButton">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>370</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>370</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>431</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Reservations With Conflicts :</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="Icons.qrc"/>
 </resources>
 <connections/>
</ui>
